update README to clarify hash functionality not needed (#6)
TheOdinProject/custom_enumerable_project: README.md

### Methods

> The following table has the methods you will need to implement along with their associated specs, as outlined in the project instructions. The methods in the Enumerable module do not need to work with non-Arrays, but you can challenge yourself and try to make them work for Hashes as well.

Reviewer: ```suggestion
The following table has the methods you will need to implement along with their associated specs, as outlined in the project instructions. The methods in the Enumerable module only need to work with Arrays, but you can challenge yourself and try to make them work for Hashes as well.
```
Great idea, but I think it flows a little better without the slight double negative. What do you think?
